daemon/volumes.go

```diff
@@ -385,15 +385,14 @@ func copyExistingContents(source, destination string) error {
 // copyOwnership copies the permissions and uid:gid of the source file
 // into the destination file
 func copyOwnership(source, destination string) error {
-	var stat syscall.Stat_t
-
-	if err := syscall.Stat(source, &stat); err != nil {
+	stat, err := system.GetStat(source)
```

Comment:
@gewoonrik If @cpuguy83 does not come back saying "let's use Lstat", then I guess we don't have a better option for now other than renaming Stat to Stat_t, and rename GetStat() to Stat().